website/docs/users/sample-data/dev-db-setup.md

```diff
@@ -0,0 +1,49 @@
+# Setup Development OMOP Database
```

Comment:
I would use the term "sample" rather than development, as for some users it may be for testing assessing the software, not developing it.